<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stereo spread</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="stereoSpread">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Detune</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="detune">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Octave spread</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="octaveSpread">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Number of osc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="numosc">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>OSC Groups</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="g1Type">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sawtooth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="g1Vol">
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="g1Octave">
           <property name="minimum">
            <number>-3</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>G1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="g2Type">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sawtooth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="g2Vol">
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="g2Octave">
           <property name="minimum">
            <number>-3</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>G2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="g3Type">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sawtooth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="g3Vol">
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="g3Octave">
           <property name="minimum">
            <number>-3</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>G3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="g4Type">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sawtooth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="g4Vol">
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="g4Octave">
           <property name="minimum">
            <number>-3</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Volume adsr</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Attack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volAttack">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Decay</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volDecay">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Sustain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volSustain">
           <property name="value">
            <number>99</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Release</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volRelease">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cutoff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="filterCutoff">
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Resonance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="filterResonance">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
